src/index.ts

```diff
@@ -326,6 +330,28 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (include && legacy) {
+      function renameImportPlugin() {
+        return {
+          name: 'rename-import-plugin',
+          resolveId(source) {
+            if (source.startsWith('/web_modules/')) {
+              return {
+                id: source.replace(/^\/web_modules\//, path.join(cwd, 'node_modules', path.sep)),
+              };
+            }
+            return null;
+          },
+        };
+      }
+      const noModuleBundle = await rollup.rollup({
+        input: glob.sync(include),
+        plugins: [renameImportPlugin(), ...inputOptions.plugins],
+      });
+      await noModuleBundle.write({file: legacy, format: 'iife'});
```

Comment:
`iife` format, with single output (specified by CLI flag, which again, can be called something else)